src/directives/repeat-click.js

```diff
@@ -1,12 +1,30 @@
 import { once, on } from 'element-ui/src/utils/dom';
 
+function getRepeatDuration(bindingArg, vnode, defaultDuration) {
+  let duration = defaultDuration;
+  if (bindingArg) {
+    let arg = bindingArg;
+    const dynamicArgMatch = typeof arg === 'string' ? /^\[(.*)\]$/.exec(arg) : null;
+    if (dynamicArgMatch) {
+      arg = vnode.context[dynamicArgMatch[1]];
+    }
+    duration = typeof arg === 'string' ? parseInt(arg, 10) : arg;
+    if (isNaN(duration)) {
+      console.warn(`Argument '${bindingArg}' could not be parsed as an integer.`);
+      duration = 100;
+    }
+  }
+  return duration;
+}
+
 export default {
   bind(el, binding, vnode) {
     let interval = null;
     let startTime;
+    let duration = getRepeatDuration(binding.arg, vnode, 100);
```

Comment:
Make the repeat interval dynamic based on the directive argument (with 100 used as the default).